app/controllers/provider/admin/service_discovery/services_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Provider::Admin::ServiceDiscovery::ServicesController < Provider::Admin::BaseController
+  before_action :autorize_create, only: :create
+  before_action :autorize_update, only: :update
+
+  load_and_authorize_resource :service, through: :current_user,
+    through_association: :accessible_services, only: %i[update]
+
+  def create
+    if can_create?
+      @service = ::ServiceDiscovery::ImportClusterDefinitionsService.create_service(current_account, cluster_namespace: create_service_params[:namespace],
+                                                                                                     cluster_service_name: create_service_params[:name])
+      flash[:notice] = 'The service will be imported shortly. You will receive a notification when it is done.'
+      redirect_to admin_services_path
+    else
+      flash[:error] = "Couldn't create service. Check your Plan limits"
```

Comment:
I suppose on premises any tenant can create any number of services.
Then leaving that does not make sense.